src/commands/templates/discover.js

```diff
@@ -120,7 +120,8 @@ class DiscoverCommand extends BaseCommand {
 
     try {
       const searchCriteria = {
-        statuses: ['Approved']
+        statuses: ['Approved'],
+        categories: ['!helper-template']
```

Comment:
let's use a dedicated constant, you need to import it first
```
const { SEARCH_CRITERIA_FILTER_NOT } = require('@adobe/aio-lib-templates')
```
then
```suggestion
        categories: [SEARCH_CRITERIA_FILTER_NOT + 'helper-template']
```

Reply:
Done ✅